egs/babel/s5d/local/lexicon/apply_morphs.sh

```diff
@@ -0,0 +1,32 @@
+#! /bin/bash
+# Copyright 2016  Johns Hopkins University (Author: Matthew Wiesner)
+# Apache 2.0
+
+encoding="utf_8"
+max_len=3
+
+. ./utils/parse_options.sh
+
+command -v morfessor-segment >/dev/null 2>&1 || { echo >&2 "Morfessor seems to either not be installed or not on path."; exit 1; }
+
+if [ $# -ne 3 ]; then
+  echo "Usage: ./local/apply_morphs.sh [opts] <morphs_model_dir> <wordlist> <_word2baseform>"
+  echo "  --encoding <encoding_type> # lexicon encoding"
+  echo "  --max-len <int> # maximum allowed morpheme length"  
+  exit 1
```

Comment:
generally speaking, when printing error mesages, the echos should be 
```
echo >&2 "something"
```
typically, we are not extremely careful about it but I think we perhaps should be.